corehq/ex-submodules/pillowtop/pillow/interface.py

```diff
@@ -206,7 +206,8 @@ def __record_change_metric_in_datadog(self, metric, change, timer=None):
                 u'document_type:{}'.format(change.metadata.document_type),
                 u'domain:{}'.format(change.metadata.domain),
                 u'is_deletion:{}'.format(change.metadata.is_deletion),
-                u'pillow_name:{}'.format(self.get_name())
+                u'pillow_name:{}'.format(self.get_name()),
+                u'publist_timestamp:{}'.format(change.metadata.publish_timestamp),
```

Comment:
This won't work - you need a new guage metric which records the timestamp as an epoch